adala/runtimes/_litellm.py

```diff
@@ -153,20 +153,32 @@ def _get_usage_dict(usage: Usage, model: str) -> Dict:
     return data
 
 
-class InstructorClientMixin:
+def resolve_litellm_model_and_provider(model_name: str, provider: str):
```

Comment:
nit: imo it's a bit clearer to name this "normalize_litellm_model_and_provider", wdyt?

Reply:
updated